iscsi/targetd/kubernetes/iscsi-provisioner-pvc-win.yaml

```diff
@@ -0,0 +1,14 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: myclaim
+  annotations:
+    volume.beta.kubernetes.io/storage-class: "iscsi-targetd-vg-targetd"
```

Comment:
why put storageclass inside annotations?

Reply:
Thanks for your review @jingxu97 
The pattern is from https://github.com/kubernetes-incubator/external-storage/blob/master/iscsi/targetd/kubernetes/iscsi-provisioner-pvc.yaml. It could be from a sample from an earlier version of k8s. Let me fix that.